### PR TITLE
add info about relative paths to config page

### PR DESCRIPTION
**Pull request by bcoca**
##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

config

### CODE REVIEW

File: docs/templates/config.rst.j2 [hunk 1]
```diff
@@ -74,6 +74,12 @@ working directory, you can explicitly specify the config file via the
 appropriate steps to mitigate the security concerns above before doing so.
 
 
+Relative paths
+--------------
+
+Many configuration options allow for relative paths being specified, in most of those cases the path used will be relative to the ansible.cfg file used for the current execution. If you need a path relative to your current working directory you can use the `{{CWD}}` macro to specify such, this is not recommended as usind your CWD can be a security risk. i.e ``cd /tmp; secureinfo=./newrootpassword ansilbe-playbook ~/safestuff/change_root_pwd.yml``.
```
Comment (Akasurde) [inline docs/templates/config.rst.j2]:
```suggestion
Many configuration options allow for relative paths being specified, in most of those cases the path used will be relative to the ansible.cfg file used for the current execution. If you need a path relative to your current working directory you can use the ``{{CWD}}`` macro to specify such, this is not recommended as using your CWD can be a security risk. i.e ``cd /tmp; secureinfo=./newrootpassword ansible-playbook ~/safestuff/change_root_pwd.yml``.
```

**Review by Akasurde — APPROVED** [review]
LGTM

**Comment by acozine** [comment]
I'm surprised this is passing Shippable - don't we have a test for line length?

**Comment by acozine** [comment]
@bcoca what's the motivation for adding this to the docs? I've broken up the line and done some tidying, but I'd like to know more before merging.

**Comment by bcoca** [comment]
@acozine to document how ` setting = ./mypath` works from configuration files, it is not specified anywhere currently.

**Pull request merged**